web-tier/src/main/java/com/outdoor/foodcalc/model/dish/DishProduct.java

```diff
@@ -1,25 +1,18 @@
 package com.outdoor.foodcalc.model.dish;
 
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
 import javax.validation.constraints.Min;
 import javax.validation.constraints.PositiveOrZero;
 import java.util.Objects;
 
-public class DishProduct {
+@AllArgsConstructor
+@NoArgsConstructor
+public @Data class DishProduct {
```

Comment:
I prefer to put class level annotation above the class definition line
```
@Data
public class DishProduct {

Reply:
done